Library/Homebrew/cmd/audit.rb

```diff
@@ -229,6 +230,11 @@ def audit_formula_name
       return
     end
 
+    if !!FORMULA_RENAMES[name]
+      problem "Formula name conflicts with renamed formula old name."
```

Comment:
This might be more informative as something like `\"#{name}\" is reserved as the old name of #{FORMULA_RENAMES[name]}`.